### DISCUSSION TITLE
courses/ts/%EB%AC%B8%EB%B2%95/01-%EA%B0%92%EC%9D%98-%ED%83%80%EC%9E%85%EC%9D%B4-%EA%B2%B0%EC%A0%95%EB%90%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EC%8B%9D

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#68] [Q&A]
# courses/ts/%EB%AC%B8%EB%B2%95/01-%EA%B0%92%EC%9D%98-%ED%83%80%EC%9E%85%EC%9D%B4-%EA%B2%B0%EC%A0%95%EB%90%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EC%8B%9D

https://lms.vanillacoding.co/courses/ts/%EB%AC%B8%EB%B2%95/01-%EA%B0%92%EC%9D%98-%ED%83%80%EC%9E%85%EC%9D%B4-%EA%B2%B0%EC%A0%95%EB%90%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EC%8B%9D

**Comment by Farmer15** [comment]
위에 타입 단언의 조건 파트에서 "더 구체적이거나 덜 구체적인 타입" 링크가 제대로 작동하지 않고 404 에러가 발생합니다~!

**Reply by oddmj** [reply]
수정했습니다. 감사합니다! 🙏
